config.py

```diff
@@ -1,7 +1,7 @@
 import os
-import urlparse
+import urllib.parse
 
-from raygun4py.middleware import flask as flask_raygun
+#from raygun4py.middleware import flask as flask_raygun
```

Comment:
Sorry, had to comment that in order to try the project, don't have Raygun account.